### DISCUSSION TITLE
(deleted spam)

### DISCUSSION THREAD

**Discussion by 521xueweihan** [Atlas-OS/Atlas#1237] [Development]
(deleted spam)

**Comment by pencilnav** [comment]
@521xueweihan 
1. You're self advertising on "Development".
2. You've created a profile page without our permission.
3. Your site serves a similar purpose to Trustpilot, which we already have.
4. You've created issues & discussions on 50 repositories using the same spamming template.
![img](https://github.com/user-attachments/assets/0d463542-b221-4e81-b00b-fcb00a3e6a1c)

# Delete the profile page that was created without our permission.

### I have also reported your spamming and abusive behavior to GitHub, and I encourage anyone seeing this to do so.

**Reply by Ast3risk-ops** [reply]
@PencilNavigator it's actually most similar to ProductHunt, look it up if you haven't seen it.

**Comment by 521xueweihan** [comment]
@PencilNavigator I have no intention of harassing others. I just want to tell the maintainers of the project that I have made efforts to promote your project. I have not done anything to damage the image of the project, so I don't know where your disgust comes from.

**Reply by Ast3risk-ops** [reply]
It's more of a problem of association. People *create* ProductHunt pages, ProductHunt doesn't do that for them. The project never asked to be associated with a small chinese producthunt knockoff (largest project on there only has around 1.56k votes and it's related to `nginx`)

Also, you're spamming repositories to promote your knockoff, and Atlas wants no association with that.

**Reply by 521xueweihan** [reply]
1. I didn't ask them must to wear the badge, the decision is up to the project maintainer
2. The data in the project details page are all public data, and there is no act of impersonating the author
3. You think I'm small, thank you for your encouragement. But I'm not a copycat, if you have used it

**Reply by 521xueweihan** [reply]
May I ask if the IMDB movie review website needs to obtain authorization from the movie before creating a profile page?

I hope you can take a look at GitHub's traffic data to see how much traffic comes from HelloGitHub recently. If you turn a blind eye to my promotion and publicity for your project, I have nothing to say.

**Comment by the-P1neapple** [comment]
@521xueweihan Thank you for your interest in our project. We as Project Maintainers have respectfully declined your offer.
